apps/web/components/email-list/EmailList.tsx

```diff
@@ -369,11 +370,15 @@ export function EmailList(props: {
   const onArchiveBulk = useCallback(async () => {
     toast.promise(
       async () => {
+        const listOfSelectedThreadId = [];
```

Comment:
I prefer to use filter here. Instead of for + push.

Reply:
How do you suggest we manage a `setIsArchiving((s) => ({ ...s, [threadId]: true }));` Archiving loader?